src/csharp/Extensions/Microsoft.Spark.Extensions.FileSystem.E2ETest/Microsoft.Spark.Extensions.FileSystem.E2ETest.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IsPackable>false</IsPackable>
```

Comment:
Why are we setting this to false?

Reply:
Hmmm, I just copied this from the other extension `E2ETest` projects.  I will remove it everywhere.